go-controller/pkg/util/gateway-cleanup.go

```diff
@@ -78,25 +79,18 @@ func GatewayCleanup(nodeName string, nodeSubnet *net.IPNet) error {
 	if err != nil {
 		return err
 	}
-	if k8sNSLbTCP != "" {
-		_, stderr, err = RunOVNNbctl("lb-del", k8sNSLbTCP)
-		if err != nil {
-			return fmt.Errorf("failed to delete Gateway router %s's TCP load balancer %s, stderr: %q, "+
-				"error: %v", gatewayRouter, k8sNSLbTCP, stderr, err)
-		}
-	}
-	if k8sNSLbUDP != "" {
-		_, stderr, err = RunOVNNbctl("lb-del", k8sNSLbUDP)
-		if err != nil {
-			return fmt.Errorf("failed to delete Gateway router %s's UDP load balancer %s, stderr: %q, "+
-				"error: %v", gatewayRouter, k8sNSLbUDP, stderr, err)
-		}
+	protoLBMap := map[kapi.Protocol]string{
```

Comment:
iirc there was one place where the unit tests would randomly fail. It was because the unit test expected order tot he nbctl commands and using a map did not guarantee order. I wonder if this was the place.

Reply:
Don't think so, because the only test that runs `lb-del` is gateway-init_test.go and that only does TCP explicitly.